client-src/elements/chromedash-drawer.js

```diff
@@ -235,29 +230,41 @@ export class ChromedashDrawer extends LitElement {
     return drawer.open;
   }
 
+  renderNavItem(url, label) {
+    return html`<a
+      href="${url}"
+      ?active=${this.isCurrentPage(url)}
+      >${label}</a>`;
+  }
+
   renderDrawer() {
     let accountMenu = nothing;
     if (IS_MOBILE && !this.loading) {
-      accountMenu = html`
-      ${this.renderAccountMenu()}
-      `;
+      accountMenu = this.renderAccountMenu();
+    }
+
+    let adminMenu = nothing;
+    if (!this.loading && this.user?.is_admin) {
+      adminMenu = this.renderAdminMenu();
```

Comment:
I think you can replace this with just
```suggestion
    let adminMenu = this.renderAdminMenu();
```
Since `loading` seems to just guard whether `this.user` is set, and `renderAdminMenu()` checks `this.user?.is_admin` itself.